app/controllers/feedback_messages_controller.rb

```diff
@@ -30,8 +30,12 @@ def create
 
   private
 
+  def recaptcha_disabled?
+    SiteConfig.recaptcha_site_key.blank? && SiteConfig.recaptcha_secret_key.blank?
```

Comment:
What happens if one of these is set and not the other? Does recaptcha need both?

Reply:
Yeah, it needs both. The `site_key` is used to display the tag, and the `secret_key` is used to validate the request.

Reply:
So should this be `SiteConfig.recaptcha_site_key.blank? || SiteConfig.recaptcha_secret_key.blank?`